tests/lean/run/grind_ignore_impl_detail.lean

```diff
@@ -3,3 +3,25 @@ example {p q : Prop} : True := by
     fail_if_success grind -- should fail because `__x` is an implementation detail, and `grind` ignores them.
     cases __x; grind
   grind
+
+example {p q : Prop} : True := by
```

Comment:
Afaict, this is the only existing test which checks the `__x` convention, so I added the examples here. I could also make a new test, though.

Reply:
Yes, let's create a new test file, maybe `tests/lean/run/implementationDetailBinder.lean`?